client/mm/mm_simple_arb.go

```diff
@@ -38,6 +36,18 @@ type SimpleArbConfig struct {
 	BaseOptions map[string]string `json:"baseOptions"`
 	// QuoteOptions are the multi-order options for the quote asset wallet.
 	QuoteOptions map[string]string `json:"quoteOptions"`
+	// AutoRebalance set to true means that if the base or quote asset balance
+	// dips below MinBaseAmt or MinQuoteAmt respectively, the bot will deposit
+	// or withdraw funds from the CEX to have an equal amount on both the DEX
+	// and the CEX. If it is not possible to bring both the DEX and CEX balances
+	// above the minimum amount, no action will be taken. Also, if the amount
+	// required to bring the balances to equal is less than MinBaseTransfer or
+	// MinQuoteTransfer, no action will be taken.
+	AutoRebalance    bool   `json:"autoRebalance"`
+	MinBaseAmt       uint64 `json:"minBaseAmt"`
+	MinBaseTransfer  uint64 `json:"minBaseTransfer"`
+	MinQuoteAmt      uint64 `json:"minQuoteAmt"`
+	MinQuoteTransfer uint64 `json:"minQuoteTransfer"`
```

Comment:
I feel like none of these should need to be manually set. We should know when to trigger an auto-rebalance based on lot size and bot setttings. Min transfers too.

Reply:
I think some people would want to be more aggressive with rebalancing and some people less. If someone allocates 4 lots on both sides of a market for simple arb, when should this be rebalanced? For Arb+mm, if there are a few matches in one direction, and there's not enough balance for all the placements on one side of the market, maybe someone would want to rebalance ASAP, while others would want to give it some time because the market may move in the other direction. I don't think there's a one size fits all.